syntaxes/carbon.tmLanguage.json

```diff
@@ -30,9 +30,13 @@
 					"name": "keyword.experimental.carbon",
 					"match": "\\b(__continuation|__run|__await)\\b"
 				},
+				{
+					"name": "keyword.WIP.carbon",
```

Comment:
@Zockedidock  Where did you get information about `WIP` features? If this is experimental part, let's put on after it has been stable.

Reply:
They are described in the Docs I linked to.

Reply:
@Zockedidock thanks, I could see it. But they don't look like `WIP`. Could you add these features in `keyword.storage`?
(I'll re-setup groups soon)

Reply:
I would still add the WIP Keywords like private, public, etc even though they are not in the language yet because the Carbon standard Libraries and other `.carbon` files in the `carbon-lang` repository use them (It's nice for reading them).

Reply:
@Zockedidock what I mean is that the keywords in this block is officially being used in https://github.com/carbon-language/carbon-lang/blob/trunk/docs/design/README.md#access-control. 

So instead of naming as WIP, let's put inside 'storage.modifier.access' (or do you have some other good key name?)